.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+## Bug Report
+<!--
+    If you are not submitting a bug report, ignore this template and see
+      http://Chapel-Issues-Best-Practices.com  (TODO)
```

Comment:
What do we expect this webpage to say that we wouldn't want to simply say here?  I'm guessing if it's not a bug report, they should simply clear out the form and do a free-form issue any way they wish?  Can we say that without redirecting them?